lib/rucio/core/authentication.py

```diff
@@ -530,12 +530,8 @@ def validate_auth_token(token: str, *, session: "Session") -> "dict[str, Any]":
     return value
 
 
-def token_dictionary(token: models.Token):
-    return {'token': token.token, 'expires_at': token.expired_at}
-
-
```

Comment:
I can keep both the function and `TokenDict` (and just make `token_dictionary` return a `TokenDict` dict instead of a generic dict) but it seemed a bit unnecessary to make a function just for that. I can add it again if you prefer.

Reply:
It’s true that with hints you can be confident that the structure of the returned value will be correct, *if you’re testing for type checking*. But what about if someone accidentally does something like:

```python
return {'token': token.token, 'expires_at': datetime.utcnow()}
```

Far-fetched, I know, but hopefully it demonstrates that this function still has some value and is effectively more strict than just type checking.

Reply:
Fixed in latest revision

Reply:
And what about `lib/rucio/core/oidc.py`?